spec/Section 7 -- Response.md

```diff
@@ -58,6 +58,7 @@ the following JSON concepts:
 | Int           | Number            |
 | Float         | Number            |
 | Enum Value    | String            |
+| Any           | Arbitrary JSON    |
```

Comment:
I think `Any Value` is appropriate for the right hand side. We should avoid confusion that `Any` is to be string-serialized JSON, which this could be misinterpreted as.